### ISSUE TITLE
dapr init failed

### ISSUE THREAD

**Issue by sequoiar** [dapr/cli#740]
## Expected Behavior

dapr init should be working on master branch

## Actual Behavior

dapr init failed

## Steps to Reproduce the Problem

install 1.2.0 dapr cli, then do dapr init

```
 dapr init
\u231b  Making the jump to hyperspace...
\u274c  Downloading binaries and setting up components...
\u274c  cannot get the latest release version: https://api.github.com/repos/dapr/dapr/releases - 404 Not Found
```

## Release Note

RELEASE NOTE:

**Comment by berndverst** [comment]
Did you build the dapr binary from the master branch? Or how did you install the dapr binary?

Also, what OS and architecture are you on?

I want to make sure I can reproduce this. Thanks!

**Comment by sequoiar** [comment]
@berndverst it's dapr cli 1.2.0 binary on Ubuntu 20.04 server.

**Comment by berndverst** [comment]
I cannot reproduce this issue on a new Ubuntu 20.04 LTS VM. Here is my entire history:

```
sudo apt-get update && sudo apt-get upgrade
sudo apt-get install     apt-transport-https     ca-certificates     curl     gnupg     lsb-release
curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /usr/share/keyrings/docker-archive-keyring.gpg
echo   "deb [arch=amd64 signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/ubuntu \
  $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
sudo apt-get update
sudo docker run hello-world
wget -q https://raw.githubusercontent.com/dapr/cli/master/install/install.sh -O - | /bin/bash
dapr
sudo systemctl start docker
sudo groupadd docker
sudo usermod -aG docker $USER
exit
docker ps
dapr init
```

**Comment by berndverst** [comment]
@sequoiar is your VM ARM? As I cannot reproduce your issue I need to conclude that the problem is with your VM or network (perhaps requests to Docker Hub or Github are being filtered).

**Comment by berndverst** [comment]
You should be able to see a JSON response when you run `curl https://api.github.com/repos/dapr/dapr/releases`

**Comment by starzhou** [comment]
dapr init always faild.
![image](https://user-images.githubusercontent.com/11733741/123225411-4985f280-d505-11eb-95b9-1f7d72960135.png)

**Comment by lilongsswang** [comment]
I use a MacBook Pro OS version 11.4, and also met this issue.

**Comment by lilongsswang** [comment]
> I use a MacBook Pro OS version 11.4, and also met this issue.

and I could visit this url and respond a JSON : https://api.github.com/repos/dapr/dapr/releases

**Comment by berndverst** [comment]
@lilongsswang @sequoiar are both of you in China or connecting through a Chinese VPN (or corporate network perhaps)? I am aware of reports such as these only from users in China.

**Comment by lilongsswang** [comment]
> @lilongsswang @sequoiar are both of you in China or connecting through a Chinese VPN (or corporate network perhaps)? I am aware of reports such as these only from users in China.

Yes, I'm in China. But I could visit this site normally.

**Comment by lilongsswang** [comment]
I use command 'dapr init' installed 2 dockers, the redis and zip-kin, remained the last one. When it failed, I use 'dapr uninstall --all', and then, I could not call 'dapr init' success any more.

**Comment by berndverst** [comment]
@skyao can you help investigate this? It only affects Chinese users. Perhaps a change for separate asset URLs similar to what was done for the Powershell installer is required here.

**Comment by wcs1only** [comment]
There is a possibility that #758 fixes this.

**Comment by wcs1only** [comment]
For a temporary workaround can you try? If that works, then there is a good chance that #758 will fix this going forward:
```
dapr init --dashboard-version 0.6.0 --runtime-version 1.2.2
```

**Comment by tanvigour** [comment]
This should be fixed now, please try to validate with 1.3

**Comment by tanvigour** [comment]
if you still have issues, please create another issue. closing this for now since this seems to be resolved
